### DISCUSSION TITLE
Avatar

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#303] [Component suggestions]
<img width="480" alt="visuele weergave van de avatar component" src="https://github.com/user-attachments/assets/0f392c48-97b0-4123-92c8-d5968dd9588b">

We willen graag meer leren over de Avatar component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook, Github of screenshots. Met deze informatie kunnen we bewijzen dat de component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1727676003839209)

---

## Naam

Avatar

## Doel

Een afbeelding die een identiteit vertegenwoordigt, meestal in de vorm van een foto, illustratie, of initialen.

## Gerelateerde componenten

[Figure](https://github.com/orgs/nl-design-system/discussions/221)

## Onderzoek

Niet veel onderzoek kunnen vinden rondom de Avatar component.

[Accessibility of Profile Pictures: Alt Text and Beyond to Express Identity Online - ACM Digital Library](https://dl.acm.org/doi/abs/10.1145/3544548.3580710)
Video over een onderzoek naar de toegankelijkheid van profielfoto’s. Uit het onderzoek blijkt dat gedetailleerde afbeeldingsbeschrijvingen en door gebruikers geschreven alt-teksten schermlezergebruikers een prettigere en meer informatieve ervaring kunnen bieden bij het bekijken van profielfoto’s.

**Comment by Rozerinay** [comment]
## Amsterdam

- [Storybook](https://designsystem.amsterdam/?path=/docs/components-feedback-avatar--docs)
- [Figma](https://www.figma.com/design/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?node-id=12946-89475&node-type=canvas&t=zxz6OJEsmN1dcYjt-11)
- [GitHub](https://github.com/Amsterdam/design-system/tree/develop/packages/css/src/components/avatar)

---

### Screenshots / Varianten

**Storybook**

Default
<img width="56" alt="image" src="https://github.com/user-attachments/assets/da8ee9f7-1338-46bb-b828-3b9ec46c050f">

With picture
The Avatar can also display a photo or other image for the person. Make sure to scale the image down to around 100 pixels to prevent unnecessary data transfers.
<img width="56" alt="image" src="https://github.com/user-attachments/assets/deff719c-adaa-4811-9704-155bd621c6c6">

Fallback icon
A user icon displays if no label and image are provided.
<img width="56" alt="image" src="https://github.com/user-attachments/assets/81216e7f-a026-4ae8-a435-b6746c56e6aa">

In a header
<img width="538" alt="image" src="https://github.com/user-attachments/assets/434cc326-03bf-4a3c-a9b7-1cb5d25065f2">

---

### Notities

- Kan een label, foto of een icoon bevatten.
- Label bestaat uit initialen en bevat maximaal 2 letters.
- Kan bestaan uit verschillende background-colors.
- Is beschikbaar in 1 formaat waarvan de grootte wordt bepaald door de regelhoogte (1,6) van de font-size level-6.

**Reply by Bondt007** [reply]
Hi all, onze versie is tov deze screenshots vernieuwd op ons Storybook. De uitlijning is verbeterd.

**Comment by Rozerinay** [comment]
## Den Haag

[Figma - WIP](https://www.figma.com/design/x4RkF6BIdrNZbh7D53NTzB/%F0%9F%92%A0-NLDS---Den-Haag---Bibliotheek?node-id=423-455&node-type=canvas&t=9Rdr6Pt1c1QKBRZ6-11)

---

### Screenshots / Varianten

Text
<img width="100" alt="image" src="https://github.com/user-attachments/assets/0e6e315f-1523-46b1-8128-938dba13948f">

Photo
<img width="100" alt="image" src="https://github.com/user-attachments/assets/f3c41011-1eb6-43eb-a763-a79077e631cd">

Icon
<img width="100" alt="image" src="https://github.com/user-attachments/assets/a8f9c4c9-01a0-4ea3-ae0c-89dc0f14591b">

---

### Notities
- Alleen beschikbaar in design en niet in code.
- Kan een label, foto of een icoon bevatten.
- Label bevat maximaal 2 letters.
- Is beschikbaar in 1 formaat.

**Comment by Rozerinay** [comment]
## Utrecht

[Figma](https://www.figma.com/design/UXIHcIurAD8hyoBWx4hDBV/NLDS---Gemeente-Utrecht---Bibliotheek?node-id=423-455&node-type=canvas&t=gvvif35zxodbcKvU-11)

---

### Screenshots / Varianten

Text
<img width="100" alt="image" src="https://github.com/user-attachments/assets/d918951c-3fe4-4d5e-bb12-282508c5138b">

Photo
<img width="100" alt="image" src="https://github.com/user-attachments/assets/cc1b981a-23e9-4643-b230-d39c217b7e13">

Icon
<img width="100" alt="image" src="https://github.com/user-attachments/assets/3a66545f-3df3-4409-b6f2-848b14033539">
---

### Notities
- Alleen beschikbaar in design en niet in code.
- Kan een label, foto of een icoon bevatten.
- Label bevat maximaal 2 letters.
- Is beschikbaar in 1 formaat.

**Comment by Robbert** [comment]
## developer.overheid.nl

https://developer.overheid.nl/repositories/amsterdam-pydantic-media-type-25698

[<img width="539" alt="Screenshot 2024-10-09 at 13 17 31" src="https://github.com/user-attachments/assets/5398603e-3588-4746-89e0-4c0e25cc361c">](https://developer.overheid.nl/repositories/minbzk-amt-9632)

```html
<img class="size-16" src="https://avatars.githubusercontent.com/u/33843588?v=4" alt="Avatar van amt">
```

[<img width="884" alt="Screenshot 2024-10-09 at 13 17 20" src="https://github.com/user-attachments/assets/cc202572-42a2-45bb-994d-ddbf3127e00f">](https://developer.overheid.nl/repositories)

```html
<div class="card__body">
        <div class="card__avatar">
            <img src="https://avatars.githubusercontent.com/u/14022058?v=4" alt="Avatar van pydantic-media-type">
        </div>
        <div class="card__left card__left--full-width">…</div>
    </div>
```

**Comment by Rozerinay** [comment]
## ✨ Dit component is nu Help Wanted ✨


Help je mee hem door de Community stappen te krijgen?
[Avatar op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=avatar)
